### PR TITLE
From NDVI satellite data to timeseries

### PR DESCRIPTION
**Pull request by Marie59**
This is a tutorial on how to use a copernicus jupyterlab to retrieve NDVI data with OpenEO and then visualize the netcdf retrieved through a notebook in the Holoviz/holoviews ecosystem/jupyterlab. 
It might need some changes/improvments @yvanlebras feel free to check it out. 

I have one issue for the last part of the tutorial I am saying to download a notebook from a github, I would like to do it as it was done here https://training.galaxyproject.org/training-material/topics/climate/tutorials/pangeo-notebook/tutorial.html. 
But for now the download of the notebook is not working. Could you tell me how to do it like the example I just cited ?
Thank you !!

1. Give your pull request a **good title**
    - e.g. "Fix typo in ansible-galaxy tutorial" or "Add new transcriptomics tutorial covering a new sequencing technology"
2. Describe your changes in detail, list anything you still need some help with or things that are still TODO
3. Check that your images are allowed to be re-hosted by the GTN!
4. Still working on it? Make it a **Draft** pull request
   - Once it is ready, choose **Ready for Review**
   - Then the automated tests will run and we will know to review and merge it

### CODE REVIEW

**Comment by yvanlebras** [comment]
Will look at it before tomorrow! THANK YOU so much Marie!

**Comment by Marie59** [comment]
hmmm I can't create a test-data folder with data because this workflow does not need any data to start as it fetches data with the first tool

**Comment by hexylena** [comment]
That's fine @Marie59, you can skip it. We don't have a good way to test those sorts of tools anyway.

File: topics/ecology/tutorials/ndvi_openeo/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  - holoviews
+contributions:
+  authorship:
+    - Marie59
+  funding:
+    - fairease
+    - eurosciencegateway
+---
+
+# Introduction
```
Comment (hexylena) [inline topics/ecology/tutorials/ndvi_openeo/tutorial.md:33]:
```suggestion
```

File: topics/ecology/tutorials/ndvi_openeo/workflows/NDVI-with-OpenEO.ga [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            "tool_state": "{\"__input_ext\": \"csv\", \"chromInfo\": \"/opt/galaxy/tool-data/shared/ucsc/chrom/?.len\", \"input\": {\"__class__\": \"ConnectedValue\"}, \"mode\": {\"mode_select\": \"scratch\", \"__current_case__\": 0}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "0.0.1",
+            "type": "tool",
+            "uuid": "b0ea1c4d-46bd-4d4f-9843-97e5251c3d8e",
+            "when": null,
+            "workflow_outputs": []
+        }
+    },
+    "tags": [],
+    "uuid": "682acb50-b39b-43b2-9864-53adf1cae47d",
```
Comment (hexylena) [inline topics/ecology/tutorials/ndvi_openeo/workflows/NDVI-with-OpenEO.ga:86]: please consider adding a License to both of these (you can do it in Galaxy's workflow editor interface, the same place you added your name.)

**Review by hexylena — APPROVED** [review]

File: topics/ecology/tutorials/ndvi_openeo/workflows/NDVI-with-OpenEO.ga (hunk 2 above)
Comment (Marie59) [reply topics/ecology/tutorials/ndvi_openeo/workflows/NDVI-with-OpenEO.ga:86]: Thought I did but I forgot to save it :)

**Comment by yvanlebras** [comment]
Hey hey hey! Just tested all the jupyternotebook steps. It seems ok, but no data on my output collection... Will see tomorrow morning if it is ok to go further!

**Comment by yvanlebras** [comment]
ok, now it is ok for the ndvi/openeo side, but with holoviz, if the download of the jupyter notebook is ok, the execution is not, error message here:
```
File Load Error for Tabular_Datasets_NDVI.ipynb
Unreadable Notebook: /data/jwd01/pulsar_staging/66379480/working/jupyter/Tabular_Datasets_NDVI.ipynb <ValidationError: "Notebook could not be converted from version 1 to version 2 because it's missing a key: cells">
```

**Comment by Marie59** [comment]
> ok, now it is ok for the ndvi/openeo side, but with holoviz, if the download of the jupyter notebook is ok, the execution is not, error message here:
> ```
> File Load Error for Tabular_Datasets_NDVI.ipynb
> Unreadable Notebook: /data/jwd01/pulsar_staging/66379480/working/jupyter/Tabular_Datasets_NDVI.ipynb <ValidationError: "Notebook could not be converted from version 1 to version 2 because it's missing a key: cells">
> ```

Thanks for the test !!
Yes in the 1st description message of this PR I am saying that I don't understand to have the notebook correctly ;)
Do you have any idea ?

**Comment by yvanlebras** [comment]
It seems there is an issue with the format of the jupyter notebook... Is your sure this notebook is working? I was trying to convert it but same error message... It would be good to open it in another jupyter system and try to convert it on this jupyter instance (if it works) or something loke that...

**Comment by Marie59** [comment]
It's working when I use it on Galaxy in a jupyterlab (I bring it directly  with the "load previous notebook" inputs). I created directly from there and then dwnloaded it and put it on github. Could the download from galaxy be the issue ?

File: topics/ecology/tutorials/ndvi_openeo/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+- Manage NDVI data
+- Handle jupyterlab tools
+tags:
+  - earth-system
+  - soil
+  - NDVI
+  - timeseries
+  - csv data
+  - copernicus
+  - holoviews
```
Comment (bgruening) [inline topics/ecology/tutorials/ndvi_openeo/tutorial.md:24]:
```suggestion
tags:
  - earth-system
  - NDVI
  - timeseries
  - copernicus
  - holoviews
```

**Comment by bgruening** [comment]
@Marie59 linting is still failing :(

**Comment by Marie59** [comment]
Yes that's normal it's because of the workflow test because I can't have test-data 
Helena said I can ignore it (I don't know how to do it in the github tests)

**Comment by bgruening** [comment]
Ok. I Helana has said that lets merge it. Good luck next week :)

**Review by bgruening — APPROVED** [review]

**Pull request merged**